packages/data/lib/repositories/trakt_movies_repository_impl.dart

```diff
@@ -25,6 +25,20 @@ class TraktMoviesRepositoryImpl implements MoviesRepository {
     return await _getMovies(TraktApiPaths.comingSoonMovies, queryParameters);
   }
 
+  @override
+  Future<Iterable<dynamic>> getCast(int movieId) async {
+    final peopleApiUri = _getPeopleApiUri(movieId);
+
+    final peopleResponse =
+        await _traktApiService.get<Map<String, dynamic>>(peopleApiUri);
+
+    final cast = List<dynamic>.from(peopleResponse.data == null
+        ? {}
+        : (peopleResponse.data as Map<String, dynamic>)['cast']);
+
+    return cast;
+  }
```

Comment:
need to refactor this method. it shouldn't return `Iterable<dynamic>`. here we should execute API call and call `ResponseModelClass.fromJson`. nothing more

Reply:
Fixed.